### ISSUE TITLE
Question About Model Weight

### ISSUE THREAD

**Issue by Lingy12** [NJUNLP/x-LLM#3]
may I ask is it possible to release the model weight? 

Thank you.

**Comment by OwenNJU** [comment]
Yeah, we are uploading it to huggingface. Upon it is finished, i will let you know then

**Comment by OwenNJU** [comment]
https://huggingface.co/YunzheLv
